core/server/helpers/meta_title.js

```diff
@@ -3,9 +3,10 @@
 //
 // Page title used for sharing and SEO
 var proxy = require('./proxy'),
+    _ = require('lodash'),
     getMetaDataTitle = proxy.metaData.getMetaDataTitle;
 
 // We use the name meta_title to match the helper for consistency:
 module.exports = function meta_title(options) { // eslint-disable-line camelcase
-    return getMetaDataTitle(this, options.data.root);
+    return getMetaDataTitle(this, options.data.root, options.hash);
```

Comment:
Could you pls pass the whole `options` object?

Otherwise it's very hard to read where e.g. `options.page` is coming from.
Also, would you mind adding a simple unit test?

https://github.com/TryGhost/Ghost/blob/master/core/test/unit/data/meta/title_spec.js

Reply:
Will do! 👍

Reply:
👍